index.html

```diff
@@ -5,7 +5,8 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="css/styles.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap">
+  <!-- <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"> -->
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong|Crete Round|Inter|Roboto|Poppins">
```

Comment:
- [ ] Dear @Anand8317, I am impressed with the changes you made and the impressive design you make. I can see you have a commented code, This is against good practice. This will make your codebase messy. Please remove the commented code. Remember you can go back with your commit history to check some code You had once.

Reply:
@Tadesse-Alemayehu Thank you for highlighting this issue. Have corrected it.

Reply:
You got it, buddy 😊